corehq/apps/cloudcare/static/cloudcare/js/formplayer/apps/menus/menu_view.js

```diff
@@ -278,8 +278,10 @@ FormplayerFrontend.module("SessionNavigate.MenuList", function (MenuList, Formpl
         childView: MenuList.CaseTileView,
         initialize: function (options) {
             MenuList.CaseTileListView.__super__.initialize.apply(this, arguments);
+            var gridPolyfillPath = FormplayerFrontend.request('gridPolyfillPath');
             generateCaseTileStyles(options.tiles);
             makeInnerGridStyle(options.maxHeight, options.maxWidth, options.numEntitiesPerRow);
+            $.getScript(gridPolyfillPath);
```

Comment:
this is technically an async call. so we'll probably run into a race condition on prod. locally the script will load almost instantaneously but it's not going to do that on prod. this returns a promise so it's good to execute this and then use `.done` to ensure it's loaded before doing any logic